recipes/mfast/all/conanfile.py

```diff
@@ -61,14 +63,13 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         self.copy("licence.txt", dst="licenses", src=self._source_subfolder)
-        if self.settings.os == "Windows":
-            shutil.move(os.path.join(self.package_folder, "CMake"),
-                        os.path.join(self.package_folder, "lib", "cmake", "mFAST"))
-        os.rename(os.path.join(self.package_folder, "lib", "cmake", "mFAST", "mFASTConfig.cmake"),
-                  os.path.join(self.package_folder, "lib", "cmake", "mFAST", "mFASTTools.cmake"))
```

Comment:
I think it's not a good idea to move and rename these files around.
So I've just changed package_info method to choose the folder accordingly.